src/renderer/data/accounts/getAccount.ts

```diff
@@ -0,0 +1,121 @@
+import * as sol from '@solana/web3.js';
+import hexdump from 'hexdump-nodejs';
+// import randomart from 'randomart'
+import { LRUCache } from 'typescript-lru-cache';
+
+import { AccountInfo } from './accountsSlice';
+
+import { Net, netToURL } from '../ValidatorNetwork/validatornetworkSlice';
+
+export const BASE58_PUBKEY_REGEX = /^[1-9A-HJ-NP-Za-km-z]{32,44}$/;
+
+const HEXDUMP_BYTES = 512;
+
+// TODO: make a gloabal state of "current slot number"
+//       and then use that to use the accountsCache if that is sufficiently up to date
+//       Also use that to decide if we need to do a validator is available check or not - if we're watching changes, then we already know...
+
+// const cache = new LRU({
+//     // the number of most recently used items to keep.
+//     max: 100,
+//     // Must be a positive integer in ms, defaults to 0, which means "no TTL"
+//     ttl: 3000,
+// })
+
+const cache = new LRUCache<string, AccountInfo>({
+  maxSize: 100,
+  entryExpirationTimeInMS: 1000,
+});
+
+export const updateCache = (account: AccountInfo) => {
+  cache.set(`${account.net}_${account.pubKey}`, account);
+};
+
+export async function getAccount(
+  net: Net,
+  pubKey: string
+): Promise<AccountInfo | Error | undefined> {
+  // logger.info("getAccount", {pubKey});
+  const cachedResponse = cache.peek(`${net}_${pubKey}`);
```

Comment:
the purpose to make trivial to keep any view updated - irrespective of where - and to allow the programmer not to worry about there being load.

basically - its a comms abstraction - watch the validator staked account some time - its SOL view gets updated in both the program changes and account view - but with this cache, it doesn't do the network request for getAccount, because the info is updated from program changes often enough.

basically, there are at least 3 different ways of getting the info from the validator, and at least 2 (and soon more) ways we may display it live - and this way, the views of that info uses as few network requests as possible to show an updated and consistent value.

I'm also planning on having a multi-browser-tab cache, so we don't add extra network and parsing load when the user does that.